packages/react-components/react-slider/src/components/Slider/Slider.types.ts

```diff
@@ -37,7 +37,10 @@ export type SliderSlots = {
   };
 };
 
-type SliderCommons = {
+export type SliderProps = Omit<
+  ComponentProps<Partial<SliderSlots>, 'input'>,
+  'defaultValue' | 'onChange' | 'size' | 'value'
+> & {
   /**
    * The starting value for an uncontrolled Slider.
    * Mutually exclusive with `value` prop.
```

Comment:
@smhigley has been alphabetizing props as part of this change. Can you update these to be alphabetized?